protobuf/syft_proto/messaging/v1/message.proto

```diff
@@ -28,4 +30,8 @@ message ObjectMessage {
 
 message CommandMessage {
   syft_proto.execution.v1.ComputationAction action = 1;
+}
+
+message CommunicationMessage {
```

Comment:
Having spent some time trying to sort out terminology, I settled on `CommandMessage` instead of `ActionMessage`, since that's how PySyft was already describing the process of sending an operation to another worker. It seems like that might cover both kinds of actions though. Maybe we should add a `oneof` to `CommandMessage` so that it can contain either a `ComputationAction` or a `CommunicationAction`?

Reply:
I agree! I was not so sure what was better between what you said and what I did but seems like CommunicationMessage is really a duplication of CommandMessage

Reply:
Yeah, my thinking on this changed a bit after we talked last. I read some more of the surrounding code and realized that the `command` terminology was already in use and felt more apt.